client/my-sites/plugins/plugin-management-v2/plugins-list/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Card } from '@automattic/components';
+import TextPlaceholder from 'calypso/jetpack-cloud/sections/partner-portal/text-placeholder';
+import PluginCard from '../plugin-card';
+import PluginsTable from '../plugins-table';
+import type { PluginColumns, PluginRowFormatterArgs } from '../types';
+import type { SiteData } from 'calypso/state/ui/selectors/site-data';
+import type { ReactElement, ReactNode } from 'react';
+
+import './style.scss';
+
+interface Props {
+	selectedSite: SiteData;
+	items: Array< any >;
```

Comment:
Can we use `Array< Plugin >` here? And what do you think about renaming `items` to `plugins` here as well?